llvm/test/MC/AMDGPU/mcexpr_amd_err.s

```diff
@@ -0,0 +1,29 @@
+// RUN: not llvm-mc -triple amdgcn-amd-amdhsa %s 2>&1 | FileCheck --check-prefix=ASM %s
+
+// ASM: 20:22: error: empty max expression
```

Comment:
can you move the checks so each one is next to the relevant code? Also should use [[@LINE+1]] instead of hardcoded line numbers